config/development.json

```diff
@@ -152,6 +152,7 @@
 		"site-indicator": true,
 		"memberships": true,
 		"support-user": true,
+		"titan/upsell-email-domain-purchase": false,
```

Comment:
I have two thoughts that contradict each other here!

1. I don't know that we need a feature flag for this -- I think it's a simple enough change that we can just remove the routing quickly via a new PR. We can control the launch by waiting to get this PR deployed. This shouldn't be so disruptive that we need really extensive testing etc.
2. If we _really_ want a feature flag, I think we can just prefix it with `email`, as the changes here do still promote Google Workspace.

Reply:
1. It's a small enough change, I agree. Will have to clean up the flags later as well if we go this route. I'm fine either way. 😝 Let me know if I should remove the flag.
2. 💯